docs/source/reference/api/eland.DataFrame.es_query.rst

```diff
@@ -0,0 +1,6 @@
+eland.DataFrame.es_query
```

Comment:
pandas autogenerates these rst files, we should maybe look to do this longer term

Reply:
Agreed! I'll create an issue for this.

Reply:
Created in: https://github.com/elastic/eland/issues/160